### PR TITLE
Update documentation to include information about running Cucumber tests

### PR DESCRIPTION
**Pull request by richaagarwal**

### CODE REVIEW

**Comment by richaagarwal** [comment]
@sebworks @chosak Anything glaringly missing here?  Any new sections I should add?

File: docs/testing.md [hunk 1]
```diff
@@ -2,19 +2,26 @@
...
+```
+
+There are several options you can pass to run a particular suite of tests,
+to run a particular list of features,
+and/or to run it in "fast" mode:
 
+```sh
+gulp test:acceptance --suite=wagtail-admin
+gulp test:acceptance --specs=multi-select.feature
+tox -e acceptance-fast
```
Comment (anselmbradford) [inline docs/testing.md]: Can be replaced with `gulp test:acceptance --fast`

Comment (sebworks) [inline docs/testing.md]: So you can do both but if you call tox it's:

` tox -e acceptance-fast specs=multi-select.feature`

**Comment by sebworks** [comment]
@richaagarwal, this looks good. Will continue modify it as we evolve. It needs some examples and guidelines but lets merge. @anselmbradford, what do you thing?

File: docs/testing.md [hunk 2]
```diff
@@ -2,19 +2,37 @@
...
-Many of our browser tests have been rewritten as Python unit tests. The remaining rely on data that is not generated during testing, so please note many of these tests might fail if you do not have a production database locally. If you do, open a new Terminal window or tab and change to the project directory,
+To run browser tests, open a new Terminal window or tab and change to the project directory,
 then tell gulp to start the tests:
 
 ```sh
 gulp build
-gulp test:acceptance --suite=integration
-gulp test:acceptance --suite=content
+gulp test:acceptance
+tox -e acceptance
```
Comment (anselmbradford) [inline docs/testing.md]: Probably should put this in a note, as you don't have to call `gulp test:acceptance` AND `tox -e acceptance`, it's just two ways to run the same thing. Or just drop one of them for simplicity and have one command to rule them all.

Comment (sebworks) [inline docs/testing.md]: ok, I added the language in https://github.com/cfpb/cfgov-refresh/pull/3012/commits/3224e84b884fa360d7cd132eec5f8b56cc7a0873. @anselmbradford, if have specific language in mind jump in and do your thing.

**Pull request merged**